### ISSUE TITLE
Circulate import issues in __init__.py prevent debugging configuration in pycharm

### ISSUE THREAD

**Issue by arovai** [bids-apps/giga_connectome#135]
Hello!
I would like to play around with the code in pycharm but got stuck when setting up run/debug configuration.
What I have tried is to set the `module` field to `giga_connectome.run` and then e.g. have `-h` in the options field.
Running this and I hit a Circular Import error:
```
ImportError: cannot import name '__version__' from partially initialized module 'giga_connectome' (most likely due to a circular import)
```
Note that I have ensured that all the requirements are satisfied in my venv.
Note also that when I install the package locally (essentially: `pip install .`), then `giga_connectome` gets added to my PATH and running `giga_connectome -h` works like a charm.
So I guess I am missing something in the pycharm run/debug configuration, hence this question :-)
Thanks for you help!
AR

**Comment by htwangtw** [comment]
thanks for the feedback! I am not a pycharm user nor do I use the debugging function much so I am afraid I cannot help much here

**Comment by Remi-Gau** [comment]
@arovai 
not a pycharm user either but maybe if you can share the code snippet of what you are trying to do maybe that could give us a clue.

**Comment by htwangtw** [comment]
I mostly do dedicated unit tests and never used breakpoint on this project (hence never noticed these issues), but I had a look into the debugger and noticed two things - 
First there are circulate import issues in `__init__.py` since redefined a lot of relative paths there. It should be fine to remove these and I believe that would fit better with the current packaging recommendation. We can do a PR to resolve this.
Second thing is we do intentionally prevent the `run.py` to run as a module (see runtime error at the end of the file.). If you run other modules in the debug mode they all should run fine.

**Comment by htwangtw** [comment]
The circular import problem has been addressed in #145 
If you continue to run into issue, please reopen this issue and we can have another look.
